premake5.lua

```diff
@@ -81,10 +81,16 @@ elseif (os.istarget("linux")) then
 
    defines { "WINDOWS=0" }
    
-   buildoptions { "-std=c++17", "-I/home/kjetil/SDKs/VST_SDK/VST2_SDK" }
+   buildoptions {
+                "-std=c++17",
+                "-I/home/kjetil/SDKs/VST_SDK/VST2_SDK",
```

Comment:
What is this hard path?

Reply:
replace your username with kjetil

Reply:
Actually it's not important,you do not need to replace anything since you are going to merge the CONTENTS of vst2_sdk folder (OLD version needed!) inside the vst3sdk folder